deployments/charts/kuma/templates/cni-rbac.yaml

```diff
@@ -21,10 +21,18 @@ metadata:
 rules:
   - apiGroups: [""]
     resources:
-      - pods
       - nodes
     verbs:
       - get
+  - apiGroups: [""]
+    resources:
```

Comment:
why do we need more permissions?

Reply:
merbridge needs it for its localip controller (ie. https://github.com/merbridge/merbridge/blob/36ee27af5021bda9e3894278af5b40542e9a3dc8/controller/localip/localip.go#L40)